{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"e688dc4d-1004-4581-8d38-c501ea8ef507\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) board with: {\"id\":\"911acab4-2fb5-47b9-a166-0abf4184e114\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"b5d97c98-c704-41e5-9548-a8414015b089\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"95d051d5-ee34-476c-aede-b252a3411f2f\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"b227cd3c-0efd-4494-97f1-2d7abd9bf31b\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"693ed93f-165d-4cb0-b779-ce3ce6c16ba9\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"9589f2d4-31af-4942-b4fa-9398e62d51a6\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: oops, stack: Error: oops\n    at Object.<anonymous> (C:\\Users\\Onya\\Downloads\\NodeJS-RS-master\\src\\app.js:35:7)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (C:\\Users\\Onya\\Downloads\\NodeJS-RS-master\\src\\server.js:2:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: oops, stack: Error: oops\n    at Object.<anonymous> (C:\\Users\\Onya\\Downloads\\NodeJS-RS-master\\src\\app.js:35:7)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (C:\\Users\\Onya\\Downloads\\NodeJS-RS-master\\src\\server.js:2:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"41092f1d-83a6-4bd1-8bd8-7bf51bcabc7e\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) board with: {\"id\":\"ccf5b393-b5ae-4922-a24d-0de9282dc569\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"b80456eb-867e-4f83-ba92-a8e915194186\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"4af05302-6455-4363-b08d-9f0f1e3a811c\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"0efb56bf-485b-4488-9d89-2d3e0eeef100\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"ff7ab742-ef2e-4117-850d-04662c31f3a9\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) task with: {\"id\":\"1b85686d-888c-4821-9749-a7a8b52fa73b\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) user with: {\"id\":\"fgnf\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"StatusCode 404, Message: Couldn't find a(an) user with: {\"id\":\"fgnf\"}","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: user.save is not a function, stack: TypeError: user.save is not a function\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:32\n    at Array.forEach (<anonymous>)\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:31:11)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:350:13\n    at handleOperationResult (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:558:5)\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:277:5)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: listen EADDRINUSE: address already in use :::4000, stack: Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (net.js:1317:16)\n    at listenInCluster (net.js:1365:12)\n    at Server.listen (net.js:1451:7)\n    at Function.listen (D:\\NodeJS\\NodeJS-RS\\node_modules\\express\\lib\\application.js:618:24)\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:39:7\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:34:5)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: listen EADDRINUSE: address already in use :::4000, stack: Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (net.js:1317:16)\n    at listenInCluster (net.js:1365:12)\n    at Server.listen (net.js:1451:7)\n    at Function.listen (D:\\NodeJS\\NodeJS-RS\\node_modules\\express\\lib\\application.js:618:24)\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:39:7\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:34:5)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: listen EADDRINUSE: address already in use :::4000, stack: Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (net.js:1317:16)\n    at listenInCluster (net.js:1365:12)\n    at Server.listen (net.js:1451:7)\n    at Function.listen (D:\\NodeJS\\NodeJS-RS\\node_modules\\express\\lib\\application.js:618:24)\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:39:7\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:34:5)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: listen EADDRINUSE: address already in use :::4000, stack: Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (net.js:1317:16)\n    at listenInCluster (net.js:1365:12)\n    at Server.listen (net.js:1451:7)\n    at Function.listen (D:\\NodeJS\\NodeJS-RS\\node_modules\\express\\lib\\application.js:618:24)\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:41:7\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:36:5)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: listen EADDRINUSE: address already in use :::4000, stack: Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (net.js:1317:16)\n    at listenInCluster (net.js:1365:12)\n    at Server.listen (net.js:1451:7)\n    at Function.listen (D:\\NodeJS\\NodeJS-RS\\node_modules\\express\\lib\\application.js:618:24)\n    at D:\\NodeJS\\NodeJS-RS\\src\\server.js:41:7\n    at NativeConnection.<anonymous> (D:\\NodeJS\\NodeJS-RS\\src\\server.js:36:5)\n    at Object.onceWrapper (events.js:420:28)\n    at NativeConnection.emit (events.js:314:20)\n    at _setClient (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:950:8)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongoose\\lib\\connection.js:797:7\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\utils.js:674:5\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\mongo_client.js:225:7\n    at connectCallback (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:366:5)\n    at D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\operations\\connect.js:602:5\n    at connectHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:297:43)\n    at cb (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:681:26)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\NodeJS\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: app.use() requires a middleware function, stack: TypeError: app.use() requires a middleware function\n    at Function.use (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\application.js:210:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:4)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:13:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:37:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:25:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:11:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:13:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:45:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: TypeError: Cannot destructure property '_id' of 'user' as it is null.\n    at Function.userSchema.statics.toResponse (D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.model.js:30:16)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:31:44\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:39:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:57:22\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:27:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:47:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:38:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:48:22\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:22:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:13:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:13:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: TypeError: Cannot destructure property '_id' of 'user' as it is null.\n    at Function.userSchema.statics.toResponse (D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.model.js:30:16)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:31:44\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:37:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:25:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:45:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:11:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:39:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:47:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:27:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:38:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:48:22\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:57:22\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:55:16\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:22:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: logg is not defined, stack: ReferenceError: logg is not defined\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:19:8)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled exception: Cannot destructure property 'method' of 'request' as it is undefined., stack: TypeError: Cannot destructure property 'method' of 'request' as it is undefined.\n    at DerivedLogger.logger.url (D:\\3 курс\\NodeJS-RS\\src\\common\\logger.js:31:11)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\app.js:20:15)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Module.require (internal/modules/cjs/loader.js:903:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (D:\\3 курс\\NodeJS-RS\\src\\server.js:3:13)\n    at Module._compile (internal/modules/cjs/loader.js:1015:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)\n    at Module.load (internal/modules/cjs/loader.js:879:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:724:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at internal/main/run_main_module.js:17:47","level":"\u001b[31merror\u001b[39m"}
{"message":"Unhandled promise rejection: user is not allowed to do action [dropDatabase] on [Rs.], stack: MongoError: user is not allowed to do action [dropDatabase] on [Rs.]\n    at MessageStream.messageHandler (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\connection.js:268:20)\n    at MessageStream.emit (events.js:314:20)\n    at processIncomingData (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\3 курс\\NodeJS-RS\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at doWrite (_stream_writable.js:403:12)\n    at writeOrBuffer (_stream_writable.js:387:5)\n    at MessageStream.Writable.write (_stream_writable.js:318:11)\n    at TLSSocket.ondata (_stream_readable.js:719:22)\n    at TLSSocket.emit (events.js:314:20)\n    at addChunk (_stream_readable.js:298:12)\n    at readableAddChunk (_stream_readable.js:273:9)\n    at TLSSocket.Readable.push (_stream_readable.js:214:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:11:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:13:34\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: TypeError: Cannot destructure property '_id' of 'user' as it is null.\n    at Function.userSchema.statics.toResponse (D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.model.js:30:16)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:31:44\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:13:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:39:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:27:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:57:22\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:37:24\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:38:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:47:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:25:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:45:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:55:16\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:22:10\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:48:22\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:11:34\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:13:34\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:13:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:37:24\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:39:24\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: TypeError: Cannot destructure property '_id' of 'user' as it is null.\n    at Function.userSchema.statics.toResponse (D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.model.js:30:16)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:31:44\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:27:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:57:22\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\tasks\\task.router.js:47:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:45:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:25:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\boards\\board.router.js:55:16\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at runNextTicks (internal/process/task_queues.js:66:3)\n    at processImmediate (internal/timers.js:434:9)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:22:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.json (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:264:10)\n    at ServerResponse.send (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:158:21)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:38:10\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
{"message":"Internal Server Error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (_http_outgoing.js:526:11)\n    at ServerResponse.header (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:771:10)\n    at ServerResponse.contentType (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:599:15)\n    at ServerResponse.sendStatus (D:\\3 курс\\NodeJS-RS\\node_modules\\express\\lib\\response.js:357:8)\n    at D:\\3 курс\\NodeJS-RS\\src\\resources\\users\\user.router.js:48:22\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async D:\\3 курс\\NodeJS-RS\\src\\errors\\catchErrors.js:5:12","level":"\u001b[31merror\u001b[39m"}
